.github/workflows/avm.utl.types.avm-common-types.yml

```diff
@@ -0,0 +1,90 @@
+name: "avm.ptn.types.avm-common-types"
+
+on:
+  workflow_dispatch:
+    inputs:
+      staticValidation:
+        type: boolean
+        description: "Execute static validation"
+        required: false
+        default: false # true
+      deploymentValidation:
+        type: boolean
+        description: "Execute deployment validation"
+        required: false
+        default: true
+      removeDeployment:
+        type: boolean
+        description: "Remove deployed module"
+        required: false
+        default: true
+      customLocation:
```

Comment:
Wondering if we want to keep the same input patameters

Reply:
Fair question. They're not used - but - I don't think we'll get around keeping them as I believe our `avm.template.module.yml` file relies on the values being passed through. Otherwise I'd have suggested to comment them out - but that won't do us any good either :/

Reply:
I'll think about that. Cool for now

Reply:
I agree. Maybe GitHub has also released new features we could use in our workflows since we last worked on them that could help us here.